dynawo/sources/Solvers/Common/DYNSolverImpl.cpp

```diff
@@ -493,9 +493,7 @@ void Solver::Impl::setSolverCommonParameters() {
   const ParameterSolver& minimumModeChangeTypeForAlgebraicRestoration = findParameter("minimumModeChangeTypeForAlgebraicRestoration");
   if (minimumModeChangeTypeForAlgebraicRestoration.hasValue()) {
     std::string value = minimumModeChangeTypeForAlgebraicRestoration.getValue<string>();
-    if (value == "DIFFERENTIAL")
```

Comment:
Algebraic restoration is normally done when there is a discontinuity in the system, demanding to reinitialize the whole system. In case of a differential mode, there is not such a discontinuity but only a change in the dynamics. So normally speaking not possible.